spanner/spannertest/README.md

```diff
@@ -12,6 +12,19 @@ production Cloud Spanner. They are reasonable for building tools, or writing
 unit or integration tests. Full-scale performance testing or serious workloads
 should use the production Cloud Spanner instead.
 
+## Maintenance Status
+
+This package is currently in maintenance mode. While it is still available for use, it is not actively maintained by the core team. We welcome external contributions and will assist with code reviews for pull requests that improve the package.
```

Comment:
Would this maybe be a bit more accurate:

```suggestion
This package is currently in limited maintenance mode. While it is still available for use, it is not actively maintained by the core team. We welcome external contributions and will assist with code reviews for pull requests that improve the package.
```

Adding 'limited' might make it a bit clearer, as now it  basically reads 'It is in maintenance mode, we are not maintaining it'.